2023-05-17 23:53:02,964 - instabot version: 0.117.0 (bot) - INFO - Instabot version: 0.117.0 Started
2023-05-17 23:53:02,964 - instabot version: 0.117.0 (bot) - DEBUG - Bot imported from C:\Users\USER\anaconda3\envs\env\lib\site-packages\instabot\bot\bot.py
2023-05-17 23:53:02,964 - instabot version: 0.117.0 (api_login) - INFO - Not yet logged in starting: PRE-LOGIN FLOW!
2023-05-17 23:53:03,318 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/get_prefill_candidates/ returned response: <Response [200]>
2023-05-17 23:53:03,673 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: qe/sync/ returned response: <Response [200]>
2023-05-17 23:53:04,019 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: launcher/sync/ returned response: <Response [200]>
2023-05-17 23:53:04,336 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/contact_point_prefill/ returned response: <Response [200]>
2023-05-17 23:53:05,780 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/login/ returned response: <Response [400]>
2023-05-17 23:53:05,780 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message":"challenge_required","challenge":{"url":"https://i.instagram.com/challenge/AXEFO31yvWv8m6GVCxgnuBXrBdKrTwtWo1COnvhdOeSWkANZXZX14u9G9CRFnU7-_e_YQ5s/Afyom8ZIx3yeFpGXKXFQu7oLNyAOJkrQJ22uv5xfGCmhB_j05sGKjvyt0QzM4GthC0EJEkvttKbMWA/","api_path":"/challenge/AXEFO31yvWv8m6GVCxgnuBXrBdKrTwtWo1COnvhdOeSWkANZXZX14u9G9CRFnU7-_e_YQ5s/Afyom8ZIx3yeFpGXKXFQu7oLNyAOJkrQJ22uv5xfGCmhB_j05sGKjvyt0QzM4GthC0EJEkvttKbMWA/","hide_webview_header":true,"lock":true,"logout":false,"native_flow":true,"flow_render_type":0},"status":"fail","error_type":"checkpoint_challenge_required"}'
2023-05-17 23:53:05,780 - instabot version: 0.117.0 (api) - ERROR - Request returns 400 error!
2023-05-17 23:53:05,781 - instabot version: 0.117.0 (api) - INFO - Instagram's error message: challenge_required
2023-05-17 23:53:05,781 - instabot version: 0.117.0 (api) - INFO - Error type: checkpoint_challenge_required
2023-05-17 23:53:07,937 - instabot version: 0.117.0 (api_photo) - ERROR - Photo Upload failed with the following response: <Response [403]>
2023-05-17 23:53:07,938 - instabot version: 0.117.0 (bot_photo) - INFO - Photo 'test2.jpg' is not uploaded.
